flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/reader/KafkaPartitionSplitReader.java

```diff
@@ -205,11 +201,9 @@ public void handleSplitsChanges(SplitsChange<KafkaPartitionSplit> splitsChange)
 
         // Seek on the newly assigned partitions to their stating offsets.
         seekToStartingOffsets(
-                partitionsStartingFromEarliest,
-                partitionsStartingFromLatest,
-                partitionsStartingFromSpecifiedOffsets);
+                partitionsStartingFromEarliest, partitionsStartingFromSpecifiedOffsets);
         // Setup the stopping offsets.
-        acquireAndSetStoppingOffsets(partitionsStoppingAtLatest, partitionsStoppingAtCommitted);
+        acquireAndSetStoppingOffsets(partitionsStoppingAtCommitted);
```

Comment:
nit: with this method only acquiring committed offsets, we should change its name.